### ISSUE TITLE
working with inner table

### ISSUE THREAD

**Issue by survivant** [ashl1/datatables-rowsgroup#6]
I have a demo that I used with datatables with inner table using ajax. 

I have rows. On the first columns of each rows I add a collapse/expand sign.  When I click on that link, it do a new ajax call and it will create a inner table (using the same layout at the main table) and this table will be insert after a row group.  

The problem is the row containing the inner table dissapears once the rowgroup kick in. and if I put this line in comment in my inner table declaration, it will works : // rowgroup[0,1]

I'll try to make a demo that is not using proprietary code.

**Comment by ashl1** [comment]
Yes. Please provide the code produced the error. You might use https://jsfiddle.net or something similar.
